app/controllers/mass_rollouts_controller.rb

```diff
@@ -52,7 +52,12 @@ def destroy
   def create_all_stages
     _, missing_stages = stages_for_creation
     missing_stages.map do |template_stage|
-      create_stage_with_group template_stage
+      begin
+        create_stage_with_group template_stage
```

Comment:
Here is the real issue. This is a feature they use only once every couple months. But Samson changes on a daily basis. So every time they've tried to use this feature, its been broken by intermediate changes in samson. 

Worse, each times, its only been some of the stages that were broken. The other stages would have been just fine, if it only had plowed through, and created them, instead of stopping when it hits the first error. So this is a feature I meant to add back with pod13.

It seems like a good compromise to me. Create what can be created so Ops can keep moving. Fix and create the rest after the initial attempt. Right now, this blocked pod 18. And if we genuinely created anything in error as a result of a bug. Then there is a nice flashy button to delete all the stages we created and start over.

Reply:
So no, checking for validations and guarding against those, isn't good enough here.